### PR TITLE
expose selectedComponent of MenuField, so the user can retrieve the s…

### PR DESCRIPTION
**Pull request by dcfranca**
…elected item details

### CODE REVIEW

**Comment by iBeliever** [comment]
@danielfranca Thanks for the pull request! Before I can merge it, I'll need to get your approval of using the MPL for your contributions (we're in the process of switching from the LGPL to the MPL). Please leave a comment on #361 if you approve of using that license for your contributions.

File: modules/Material/MenuField.qml [hunk 1]
```diff
@@ -46,6 +46,7 @@ Item {
     readonly property string selectedText: (listView.currentItem) ? listView.currentItem.text : ""
 
     property alias selectedIndex: listView.currentIndex
+    property var selectedComponent: { var get = model['get']; if (get) { return model.get(selectedIndex); } else { return null; } }
```
Comment (ricardomv) [inline modules/Material/MenuField.qml]: please indent the code and stick to 80 chars per line

**Comment by dcfranca** [comment]
@iBeliever Thanks, I just updated the code following the guidelines and added my comment approving the use of MPL

**Comment by dcfranca** [comment]
@iBeliever Any feedback?

**Comment by dcfranca** [comment]
@iBeliever Any news/feedback/update on this one?

**Comment by dcfranca** [comment]
ok, giving up... removing the pull request

**Pull request closed without merging**